src/test/java/org/springframework/cloud/deployer/spi/kubernetes/PropertyParserUtilsTests.java

```diff
@@ -54,6 +55,42 @@ public void testAnnotationParseMultiple() {
 		assertThat(annotations.get("annotation2").equals("value2")).isTrue();
 	}
 
+	@Test
+	public void testAnnotationParseMultipleWithCommas() {
+		Map<String, String> annotations = PropertyParserUtils.getStringPairsToMap("annotation1:\"value1,a,b,c,d\",annotation2:value2");
```

Comment:
I know that this style of AssertJ was already in this test but this can be simplified by using some of the map assert API provided by AssertJ such as: 
```java
assertThat(PropertyParserUtils.getStringPairsToMap("annotation1:\"value1,a,b,c,d\",annotation2:value2"))
    .isNotEmpty()
    .containsExactly(entry("annotation1", "\"value1,a,b,c,d\""), entry("annotation2", "value2"));
```